networkx/algorithms/bridges.py

```diff
@@ -69,7 +69,13 @@ def bridges(G, root=None):
     H = nx.Graph(G) if multigraph else G
     chains = nx.chain_decomposition(H, root=root)
     chain_edges = set(chain.from_iterable(chains))
-    for u, v in H.edges():
+    H_copy = H.copy()
+    if root:
```

Comment:
```suggestion
    if root is None:
```
Using `root=0` will not work when the if clause is `if root:`. Here we are checking whether there was no input, so we want to look explicitly at whether the root is the None object.  `if root is None:`

Reply:
@dschult 's point still stands here - this should be `if root is None`.

Reply:
Yes, my suggestion was exactly wrong (but for the right reasons),.  The code should be `if root is not None:`